src/test/cue_test.cpp

```diff
@@ -10,12 +10,8 @@ namespace mixxx {
 
 TEST(CueTest, DefaultCueInfoToCueRoundtrip) {
     const CueInfo cueInfo1;
-    const Cue cueObject(
-            cueInfo1,
-            audio::SampleRate(44100),
-            true);
-    auto cueInfo2 = cueObject.getCueInfo(
-            audio::SampleRate(44100));
+    const Cue cueObject(cueInfo1, audio::SampleRate(44100));
```

Comment:
Why did you change this formatting manually?? The arguments have been placed onto single lines intentionally and our code formatting config preserve this style.

Reply:
If this is caused by your IDE then you are responsible to prevent it doing this.

Reply:
Or just use `git add -p` and only stage the  hunks for commit that you actually want to change.

Reply:
@Holzhaus you can stop suggesting that, I almost always look at every single change before committing.

I thought that now with less arguments it might be sensible to compress this, especially as the arguments are not very interesting on their own.